networking/p2p/src/main/java/tech/pegasys/teku/networking/p2p/network/config/NetworkConfig.java

```diff
@@ -281,7 +283,23 @@ public FilePrivateKeySource(String fileName) {
     }
 
     @Override
-    public Bytes getPrivateKeyBytes() {
+    public Bytes getOrGeneratePrivateKeyBytes() {
```

Comment:
NIT: I think we should extract `PrivateKeySource` and `FilePrivateKeySource` into their own source files.